tests/core/test_configuration.py

```diff
@@ -641,11 +641,11 @@ def test_auth_backends_adds_session(self):
 
     def test_command_from_env(self):
         test_cmdenv_config = """[testcmdenv]
-itsacommand = NOT OK
-notacommand = OK
-"""
+            itsacommand=NOT OK
+            notacommand=OK
+        """
```

Comment:
```suggestion
        """)
```

Oops, almost missed this part :)

Reply:
Thanks for your time! I was going to commit to your suggestions but saw this was missing. I just saw this now you have posted :) That's why I changed it and pushed them in a single commit.